docs/build/apps/wallet/sep24.mdx

```diff
@@ -789,4 +789,4 @@ watcher.controller.stream.listen(
 [sep-10]: https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md
 [sep-24]: https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md
 [stellar authentication]: ./sep10.mdx
-[submitting transactions and handling errors gracefully]: /data/horizon/api-reference/errors/error-handling.mdx
+[submitting transactions and handling errors gracefully]: ../../..//data/apis/horizon/api-reference/errors/error-handling.mdx
```

Comment:
probably has nothing to do with the github action build error, but this double-slash should be fixed